src/assets/scripts/groepen/Civikaartje.jsx

```diff
@@ -0,0 +1,40 @@
+var React = require("react");
+var $ = require("jquery");
+var _ = require("underscore");
+
+class Civikaartje extends React.Component {
+
+  constructor(props, context) {
+    super(props);
+    this.profiel = props.profiel;
+    this.context = context;
```

Comment:
This is the way it is handled as well in MededelingenList. So please tell me whether i should change both or none